project/threads/urls.py

```diff
@@ -0,0 +1,47 @@
+from django.conf.urls import url, include
+from rest_framework.routers import DefaultRouter
+
+from api import views, read, write
+
+router = DefaultRouter(trailing_slash=False)
+router.register(r"threads", views.ThreadViewSet)
+router.register(r"categories", views.CategoryViewSet)
+router.register(r"accounts", views.ProfileViewSet)
+router.register(r"civis", views.CiviViewSet)
+
+urlpatterns = [
+    url(r"^v1/", include(router.urls)),
+]
+
+urlpatterns += [
+    url(r"^account_data/(?P<user>[-\w]+)/$", read.get_user, name="get user"),
+    url(r"^account_profile/(?P<user>[-\w]+)/$", read.get_profile, name="get profile"),
+    url(r"^account_card/(?P<user>[-\w]+)$", read.get_card, name="get card"),

```

Comment:
Accounts-related code should not be moved as part of this pull request. We are moving the accounts code in issue #1031 
```suggestion
```